admiral/pkg/clusters/handler.go

```diff
@@ -176,16 +176,25 @@ func makeVirtualService(host string, gateways []string, destination string, port
 		Http:     []*v1alpha32.HTTPRoute{{Route: []*v1alpha32.HTTPRouteDestination{{Destination: &v1alpha32.Destination{Host: destination, Port: &v1alpha32.PortSelector{Number: port}}}}}}}
 }
 
-func getDestinationRule(host string, locality string, gtpWrapper *v1.GlobalTrafficPolicy) *v1alpha32.DestinationRule {
+func getDestinationRule(host string, locality string, gtpTrafficPolicy *model.TrafficPolicy) *v1alpha32.DestinationRule {
 	var dr = &v1alpha32.DestinationRule{}
 	dr.Host = host
 	dr.TrafficPolicy = &v1alpha32.TrafficPolicy{Tls: &v1alpha32.TLSSettings{Mode: v1alpha32.TLSSettings_ISTIO_MUTUAL}}
-	if gtpWrapper != nil {
+	processGtp := true
+	if len(locality) == 0 {
+		log.Errorf(LogErrFormat, "Process", "GlobalTrafficPolicy", host, "", "Skipping gtp processing, locality of the cluster nodes cannot be determined. Is this minikube?")
+		processGtp = false
+	}
+	outlierDetection := &v1alpha32.OutlierDetection{
```

Comment:
This may be out of scope for this PR, but is there any plans to make these values configurable in any way? If not, are we sure these are the defaults we want? (100 consecutive errors seems high to me)

Reply:
Good point, the increase to 100 was more to fix the flickery behavior upon 503s but this is backward incompatible behavior change. I will switch it back to 10 until we make this configurable (a separate issue exists)